scripts/dropstat

```diff
@@ -403,15 +408,25 @@ Examples:
     # Variables
     parser.add_argument('-g', '--group',   type=str, help='The group of the target drop counter', default=None)
     parser.add_argument('-t', '--type',    type=str, help='The type of the target drop counter', default=None)
+    parser.add_argument('-n', '--namespace', type=str, help='Namespace name', default=None)
 
     args = parser.parse_args()
 
     command = args.command
 
     group = args.group
     counter_type = args.type
+    namespace = args.namespace
 
-    dcstat = DropStat()
+    load_db_config()
+    namespaces = multi_asic.get_namespace_list()
+    if namespace and namespace not in namespaces:
+        raise Exception("Input arguments error. Namespaces must be one of", *namespaces)
+
+    if multi_asic.is_multi_asic() and not namespace:
```

Comment:
this can be replaced with a more consistent way at line 411:
@click.option('-n', '--namespace', help='Namespace name',
             required=True if multi_asic.is_multi_asic() else False, type=click.Choice(multi_asic.get_namespace_list()))

Reply:
@wenyiz2021 are you sure we should be using the `@click` decorators within the `scripts/` folder?
This `dropstat` file doesn't hold any CLI commands just the implementation code for querying the redis DBs

Reply:
See latest change. I took your suggestion to use the click library. Thanks!